{"uuid":"92a81d0e-72c4-413e-b393-79e6fc2c37b8","historyId":"348511f2bbe6508e872d23c9ce715db0","fullName":"Tests.CreateTicketEmptyTrying.CreateTicket_Empty_Num_Field_Scenario","labels":[{"name":"package","value":"Tests.CreateTicketEmptyTrying"},{"name":"testClass","value":"Tests.CreateTicketEmptyTrying"},{"name":"testMethod","value":"CreateTicket_Empty_Num_Field_Scenario"},{"name":"parentSuite","value":"TestSuite"},{"name":"suite","value":"CreateTicket_Negative_Test_cases"},{"name":"subSuite","value":"Tests.CreateTicketEmptyTrying"},{"name":"host","value":"DESKTOP-FQ8REV2"},{"name":"thread","value":"3192@DESKTOP-FQ8REV2.TestNG-test-CreateTicket_Negative_Test_cases-1(33)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"CreateTicket_Empty_Num_Field_Scenario","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Not all steps passed expected:<10> but was:<6>","trace":"java.lang.AssertionError: Not all steps passed expected:<10> but was:<6>\r\n\tat org.testng.AssertJUnit.fail(AssertJUnit.java:65)\r\n\tat org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:467)\r\n\tat org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)\r\n\tat org.testng.AssertJUnit.assertEquals(AssertJUnit.java:318)\r\n\tat Tests.CreateTicketEmptyTrying.CreateTicket_Empty_Num_Field_Scenario(CreateTicketEmptyTrying.java:135)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\r\n\tat java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java)\r\n\tat --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)\r\n\tat java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:151)\r\n\tat org.testng.internal.thread.graph.TestNGFutureTask.<init>(TestNGFutureTask.java:15)\r\n\tat org.testng.internal.thread.graph.GraphOrchestrator.runNodes(GraphOrchestrator.java:58)\r\n\tat org.testng.internal.thread.graph.GraphOrchestrator.run(GraphOrchestrator.java:46)\r\n\tat org.testng.TestTaskExecutor.execute(TestTaskExecutor.java:69)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:720)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1742204850366,"stop":1742204853473}